raglweb/templates/info.html

```diff
@@ -26,10 +26,11 @@ <h2>The essentials</h2>
 <dl>
 	<dt>Game version:</dt><dd><a href="https://github.com/OpenRA/OpenRA/releases/tag/release-20210321">release-20210321</a></dd>
 	<dt>Game server instances:</dt><dd>|ragl.org| RAGL Server <i>N</i></dd>
-	<dt>Map pack:</dt><dd><a href="{{ url_for('static', filename='ragl-map-pack-2020-12-28.zip') }}">2020-12-28</a></dd>
+	<dt>Map pack:</dt><dd><a href="{{ url_for('static', filename='ragl-map-pack-2020-10-09.zip') }}">2020-10-09</a></dd>
```

Comment:
2021

Also, there is another link to that map pack at the bottom (we probably want that in the configuration at some point to reduce the risk of forgetting one).